frontend/app/styles/app.scss

```diff
@@ -192,6 +193,21 @@ li a {
   margin-left: 6px;
 }
 
+.encryptable-row-icons {
+  margin-left: auto;
+}
+
+@media (max-width: 1062px) {
+  .encryptable-row-attribute,
+  .encryptable-row-copy-button {
+    display: none !important;
+  }
+}
+
+.file-upload-button {
+  width: 17% !important;
```

Comment:
is important really needed?

Reply:
Couldn't find the button inside the browser which uses this css class. Is this even needed?

Reply:
Has been resolved.